### ISSUE TITLE
Disable rageclicks for certain elements

### ISSUE THREAD

**Issue by MarconLP** [PostHog/posthog#22257]
## Is your feature request related to a problem?

We expect rage clicks on certain events and do not want to be notified by them.

## Describe the solution you'd like

A separate class similar to `ph-no-capture` that continues to capture regular `$autocapture` clicks.

## Additional context

https://posthoghelp.zendesk.com/agent/tickets/13343

**Comment by mobob** [comment]
Yes, would like to see this.

I have notifications to Slack setup for rageclicks (amongst other things). I have this element on a calendar control...  it triggers a rageclick everytime. I haven't figured out the secret sauce of hogql either to exclude it from my Slack Destination (so would welcome pointers on that as a workaround).

```
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-chevron-right h-6 w-6" orientation="right"></svg>
```

**Comment by darkopia** [comment]
+1 https://posthoghelp.zendesk.com/agent/tickets/24240 (moved to PostHog: https://us.posthog.com/project/2/support/tickets/26393)

**Comment by jasonstitt** [comment]
Yes, we have a page with spinner inputs on it and all usage is rageclicks.

**Comment by pauldambra** [comment]
you can now add a CSS selector ignore list

and after https://github.com/PostHog/posthog-js/pull/2565 we will do some autodetection of elements not to capture as rageclick

**Comment by pauldambra** [comment]
this was shipped ages ago. you can configure to ignore rageclicks by adding `.ph-no-rageclick` or setting a custom element selector instead
